packages/cli/config/index.ts

```diff
@@ -290,6 +290,20 @@ const config = convict({
 	},
 
 	queue: {
+		health: {
+			active: {
+				doc: 'If health checks should be enabled',
+				format: 'Boolean',
+				default: false,
+				env: 'QUEUE_HEALTH_CHECK_ACTIVE',
+			},
+			port: {
+				doc: 'Port to serve health check on if activated',
+				format: Number,
+				default: 5678,
```

Comment:
Perhaps use a different default port for the health check?

Reply:
Honestly not sure if that is more helpful or confusing. Think it would be great if the n8n default port is simply always the same no matter where. Generally should the main process and the workers not run on the same machine. To avoid collisions and crashes anyway (in case somebody wants to do it for whatever reason) is it disabled by default and the port changeable.

Reply:
Agreed, it is just a minor thing I faced while testing that causes no harm.